src/main/resources/config.yml

```diff
@@ -55,6 +55,7 @@ settings:
     lore-length: 36
     past-clans-limit: 10
     username-regex: '(\.|\*){0,1}[a-zA-Z0-9_$]{1,16}'
+    date-format-patter: 'MMM dd, yy h:mm a'

```

Comment:
Change the default value to `HH:mm - dd/MM/yyyy` and the key to "date-time-format-pattern" (format or pattern could be removed tho 🤔)